api-server/src/events/dto/eventDTO.ts

```diff
@@ -65,6 +65,9 @@ export default class EventDTO {
   @ApiProperty({ example: EXAMPLE_DATE })
   createdAt: Date;
 
+  @ApiProperty({ example: false })
+  multi_day: boolean;
+
```

Comment:
^ This was a bug I found while testing. We have not been serializing multi_day when we fetch

Reply:
Is this property even used? I think it may be an old column from an earlier format.

It's definitely not the column affected by the `category` radio group (single/multi-day event, gallery show, etc).

Reply:
probably right,  this makes it work consistently so I say we leave this fix, but in another pr delete it properly from the db through the code